src/include/cryptfs_tpm2.h

```diff
@@ -163,7 +247,7 @@ extern const char *cryptfs_tpm2_build_machine;
 extern int option_quite;
 extern bool option_no_da;
 
-#define TPM_ALG_AUTO		0x4000
+#define TPM2_ALG_AUTO		0x4000
```

Comment:
Everything looks good, except this macro should be enclosed in TSS2_LEGACY_V1 and !TSS2_LEGACY_V1.

Reply:
TPM_ALG_AUTO was local macro, not from tpm2-tss. I just renamed it to TPM2_ALG_AUTO, all references to it are TPM2_ALG_AUTO, no matter it is v1 or v2 , and no TPM_ALG_AUTO anymore. So I think it is good.